locale/en/submission.po

```diff
@@ -254,6 +254,9 @@ msgstr "Date Published"
 msgid "publication.editDisabled"
 msgstr "This version has been published and can not be edited."
 
+msgid "publication.editorEditWarning"
```

Comment:
This locale key is the same across all 3 apps; it should be moved to the locale files in `pkp-lib`.